index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Product landing page</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+
+    <header id="header">
+      <nav id="nav-bar">
+          <img id="header-img" class="logo" src="/samsung.png" alt="samsung-logo">
```

Comment:
Avoid blank spaces in the index.html file.

Reply:
Nice catch Grace! 👍  Randomly placed blank spaces makes the code so unpleasant to read especially when you want to find which tag closes which. 😩. But how about we let the student know why as well. I think that when you tell the student something is wrong you might want to tell the student why it is wrong so that the student would have context and a better idea of how to fix the problem probably. We don't want to send the student on a wild goose chase, do we? 😅. 

Also, I think your comment could have been structured in a more polite and suggestive manner 😅. Check this out:
```
You have done very well on this HTML document but the randomly placed blank spaces in your code makes it a bit troublesome to read. I suggest that you keep it consistent with where you place your blank lines. For instance, you can space out sibling elements with blank lines to make your code more readable instead of placing them right after opening tags.
```
- Acknowledged that the student put in work
- Pointed out the problem
- Politely suggested a change 
- Gave an example for context and said why